yb-voyager/src/srcdb/data/gather-assessment-metadata/oracle/object-type-mapping.sqlplus

```diff
@@ -34,7 +34,29 @@ ON COMMIT PRESERVE ROWS AS
     WHERE
         owner = '&1'
         AND table_name NOT LIKE 'DR$%'
-        AND table_name NOT LIKE 'AQ$%';
+        AND table_name NOT LIKE 'AQ$%'
+    UNION ALL
+    SELECT
+        owner AS schema_name,
+        type_name AS object_name,
+        'INHERITED TYPE' AS object_type
+    FROM
+        dba_types
+    WHERE
+        owner = '&1'
+        AND supertype_name IS NOT NULL
+    UNION ALL
+    SELECT
+        owner AS schema_name,
+        table_name || '.' || column_name AS object_name,
+        'VIRTUAL COLUMN' AS object_type
```

Comment:
Does it make sense to add Virtual columns to "objects" ? It seems to be more suitable for table-columns-data-types?